internal/controller/components/modelregistry/modelregistry_support.go

```diff
@@ -30,6 +30,7 @@ var (
 		"IMAGES_REST_SERVICE":           "RELATED_IMAGE_ODH_MODEL_REGISTRY_IMAGE",
 		"IMAGES_OAUTH_PROXY":            "RELATED_IMAGE_OSE_OAUTH_PROXY_IMAGE",
 		"IMAGES_CATALOG_DATA":           "RELATED_IMAGE_ODH_MODEL_METADATA_COLLECTION_IMAGE",
+		"IMAGES_BENCHMARK_DATA":         "RELATED_IMAGE_ODH_MODEL_METADATA_COLLECTION_IMAGE",
```

Comment:
Since the image is the same and files are in a different location, this is correct to have the same RELATED_IMAGE variable be used to substitute for different IMAGES_* targets in the model registry manifest. 
When a separate image is used for benchmark data, we can update this variable in the odh operator.